360i.html

```diff
@@ -0,0 +1,111 @@
+<html>
+<head>
+	
+	<h1>360 i</h1>
+<h2>
+<div id="box">
+	<div class = "box" id = "LogIn"><a href="main.html">LOG IN</a></div></h2>
+	<h3>Lorem ipsum dolor sit amet, 
+		consectetur adipiscing elit, sed do eiusmod tempor incididunt 
+		ut labore et dolore magna aliqua. Ut enim ad minim veniam, 
+		quis nostrud exercitation</h3>
+
+
+<div class="img">
+<img src="image/background.jpg"  style= "widht:100%; height:100%"; position:absolute;top:0 z-index: -1;>
+
+	<style>
```

Comment:
Move your style to an external stylesheet-- like style.css that you then bring in via `<link rel="stylesheet" type="text/css" href="style.css">` in the `<head>`